==================================== INPUT ====================================
# expected-to-throw

fragment refetchableConflictWithOperationF on Node
  @refetchable(queryName: "refetchableConflictWithOperationQuery")
{
  __typename
}

query refetchableConflictWithOperationQuery {
  node(id: "y") {
    __typename
  }
}
==================================== ERROR ====================================
âœ–ï¸Ž A unique query name has to be specified in `@refetchable`, an operation `refetchableConflictWithOperationQuery` already exists.

  refetchable_conflict_with_operation.invalid.graphql:4:27
    3 â”‚ fragment refetchableConflictWithOperationF on Node
    4 â”‚   @refetchable(queryName: "refetchableConflictWithOperationQuery")
      â”‚                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    5 â”‚ {

  â„¹ï¸Ž an operation with that name is already defined here

  refetchable_conflict_with_operation.invalid.graphql:9:7
    8 â”‚ 
    9 â”‚ query refetchableConfl[31mi[0m[31mc[0m[31mt[0m[31mW[0m[31mi[0m[31mt[0m[31mh[0m[31mO[0m[31mp[0m[31me[0m[31mr[0m[31ma[0m[31mt[0m[31mi[0m[31mo[0m[31mn[0m[31mQ[0m[31mu[0m[31me[0m[31mr[0m[31my[0m {
      â”‚ [31m      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^[0m
[1m   10 â”‚ [0m  node(id: "y") {
